app/javascript/packages/document-capture/context/failed-capture-attempts.tsx

```diff
@@ -143,10 +145,14 @@ function FailedCaptureAttemptsContextProvider({
     incrementFailedCameraPermissionAttempts();
   }
 
-  const forceNativeCamera =
+  const failedAttemptsGreaterThanOrEqualToMaxAttempts =
```

Comment:
This is a really long name. I was having a hard time coming up with something clear and concise - open to ideas.

Reply:
That is shorter...it's not technically accurate though, since it's also `if equal` / `>=`.

`failedAttemptsExceedsOrEqualToMaxAttempts` is at least slightly shorter though...I'm on the fence! Do you have a preference or feeling of which is better:

1. `failedAttemptsExceedsMaxAttempts`: more readable but less accurate
2. `failedAttemptsExceedsOrEqualToMaxAttempts` accurate but less readable

Reply:
`hasExhaustedAttempts` is perfect! Thank you for that. Updated in 2eca1f88c81b94b3d3ab18a8402c76a31fb2bf65.